normandy/recipes/api/serializers.py

```diff
@@ -102,6 +103,43 @@ class Meta:
             'is_approved',
         ]
 
+    def validate_arguments(self, value):
+        # Get the schema associated with the selected action
+        try:
+            schema = Action.objects.get(name=self.initial_data.get('action')).arguments_schema
+        except:
+            raise serializers.ValidationError('Could not find arguments schema.')
+
+        schemaValidator = Validator(schema)
+        errorResponse = {}
+        errors = sorted(schemaValidator.iter_errors(value), key=lambda e: e.path)
+
+        for error in errors:
```

Comment:
Added a comment [here](https://github.com/brittanystoroz/normandy/blob/7558686c5c86587ca361b3b9a3011804923c8613/normandy/recipes/api/serializers.py#L117-L120)  

But could/should I also just link to the `ValidationError` docs in jsonschema? [#jsonSchema.ValidationError](http://python-jsonschema.readthedocs.io/en/v0.8.0/errors/#jsonschema.ValidationError)